src/documents/android/store/Store_IStoreAssets.html.md

```diff
@@ -110,7 +110,7 @@ public class ExampleAssets implements IStoreAssets{
   /** Virtual Categories **/
 
   public static VirtualCategory GENERAL_CATEGORY = new VirtualCategory (
-    "GENERAL", new ArrayList<String>(Arrays.asList(SHIELD_GOOD))
+    "GENERAL", new ArrayList<String>(Arrays.asList(new String[] { SHIELD_GOOD.getItemId() }))
```

Comment:
@EugeneButusov, about this one:
http://docs.oracle.com/javase/7/docs/api/java/util/Arrays.html#asList(T...)